ext/crypto/00_crypto.js

```diff
@@ -3429,6 +3431,70 @@
         // 4.
         return cipherText.buffer;
       }
+      case "AES-GCM": {
+        normalizedAlgorithm.iv = copyBuffer(normalizedAlgorithm.iv);
+
+        // 1.
+        if (data.byteLength > (2 ** 39) - 256) {
```

Comment:
Ping @littledivy - can you add those comments? Thanks.

Reply:
Irrespective of the nonce/IV, GCM limits plaintext to be <= 2 ^ 39 - 256. 
![image](https://user-images.githubusercontent.com/34997667/148191804-d2bde867-e82a-4bed-a9fd-b68bc8a57f84.png)

Reply:
Oh, you're right. It's because the internal counter is 32 bits wide, encrypts 128 or 2**7 bits each turn, therefore you can't go beyond 2**(32+7==39) without wrapping.

(The -256 is because of the tag and because the counter starts at 1, I think.)

Anyway, objection withdrawn. :-)